webapp/pages/post/_id/_slug/index.spec.js

```diff
@@ -40,38 +44,53 @@ describe('PostSlug', () => {
       },
       $apollo: {
         mutate: jest.fn().mockResolvedValue(),
+        query: jest.fn().mockResolvedValue({ data: { PostEmotionsCountByEmotion: {} } }),
       },
+      $scrollTo: jest.fn(),
+    }
+    stubs = {
+      HcEditor: { render: () => {}, methods: { insertReply: jest.fn(() => null) } },
```

Comment:
nifty

Reply:
yes, this is how we can stub the Editor and still check to see that it's methods have been called triggered by user interaction